src/lib/utilities/request-from-api.ts

```diff
@@ -68,11 +68,17 @@ export const requestFromAPI = async <T>(
   } = init;
   let { options } = init;
 
-  const nextPageToken = token ? { next_page_token: token } : {};
-  const query = new URLSearchParams({
-    ...params,
-    ...nextPageToken,
-  });
+  let query = new URLSearchParams();
+  if (params?.entries) {
+    query = params as URLSearchParams;
```

Comment:
Doesn't look like `params` is being used outside of this conditional, but might be best to clone this in order to future proof it.

```suggestion
    query = {...params} as URLSearchParams;
```

Reply:
That was issue before, spreading URLSearchParams converts them to empty object.

Reply:
TIL, thanks!